ajax/libs/goalProgress/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "goalProgress",
+  "filename": "goalProgress.min.js",
+  "version": "1.0",
+  "description": "Animated progress bar using jQuery to show how close you are to reaching your goal.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tinacious/goalProgress.git"
+  },
```

Comment:
Let's add `"homepage": "http://tinacious.github.io/goalProgress/"`.
Help this lib has more information.
Thank you.